frontends/election-manager/src/screens/ballot_screen.tsx

```diff
@@ -241,14 +276,10 @@ export function BallotScreen(): JSX.Element {
               warning={ballotMode !== BallotMode.Official}
             >
               Print {ballotCopies}{' '}
-              {ballotMode === BallotMode.Official ? (
-                'Official'
-              ) : (
-                <strong>
-                  {ballotMode === BallotMode.Test ? 'Test' : 'Sample'}
-                </strong>
-              )}{' '}
-              {isAbsentee ? <strong>Absentee</strong> : 'Precinct'}{' '}
+              <strong>
+                {ballotModeToReadableString(ballotMode)}{' '}
+                {isAbsentee ? 'Absentee' : 'Precinct'}
+              </strong>{' '}
```

Comment:
Slight change here - rather than only bolding certain ballot modes/types, I decided it couldn't hurt to always bold the type of ballot being printed. Makes the code a bit simpler, too (FYI @beausmith)

Reply:
Appreciate this simplification.